### PR TITLE
Assignment 8

### PR DESCRIPTION
**Pull request by CANCANNEED23**
Don't merge, this is for assignment 8.

### CODE REVIEW

File: relationship_diagram.jpg
Comment (navteniev) [inline relationship_diagram.jpg:1]: It's not clear if `workouts` is a list of long values, similarly it's not clear if `exercises` is a list of long values.

**Review by navteniev — APPROVED** [review]
High Level Diagram: 2/3 missing information about communication protocols
Entity Diagram: 2/3 relationships between entities are not well described
Call Sequence Diagram: 3/3 
PR: 0/1 includes changes from other work.
Final: 7 pts.

**Comment by LordFarquaadtheCreator** [comment]
looking at the files changed, it looks like your pr isn't up to date with main and will override our changes. please merge with main

**Comment by LordFarquaadtheCreator** [comment]
actually no wait this isn't a real pr - nothing new to add to main. Closing pr

**Pull request closed without merging**